client_internals_it_test.go

```diff
@@ -488,6 +490,71 @@ func proxyManagerShutdownTest(t *testing.T) {
 	require.True(t, ok)
 }
 
+func infiniteRestartTest(t *testing.T) {
+	skip.If(t, "enterprise")
```

Comment:
why skip if enterprise?

Reply:
When entprise flag is on, we create clusters with SSL enabled. This test creates a cluster with SSL disabled, so the client wouldn't be able to connect to the cluster if enterprise flag is on.